data_steward/cdr_cleaner/cleaning_rules/deid/geolocation_concept_suppression.py

```diff
@@ -0,0 +1,134 @@
+"""
+Sandbox and record suppress all records with a concept_id or concept_code relating to Geo Location information.
+
+Original Issue: DC-1385
+
+suppress all records associated with a GeoLocation identifier concepts in PPI vocabulary 
+The concept_ids to suppress can be determined from the vocabulary with the following regular expressions.
+        REGEXP_CONTAINS(concept_code, r'(SitePairing)|(City)|(ArizonaSpecific)|(Michigan)|(_Country)| \
+        (ExtraConsent_[A-Za-z]+((Care)|(Registered)))')AND concept_class_id = 'Question')
+and also covers all the mapped standard concepts for non standard concepts that the regex filters.
+"""
+
+# Python Imports
+import logging
+
+# Third Party Imports
+from google.cloud.exceptions import GoogleCloudError
+
+# Project Imports
+from common import OBSERVATION
+from common import JINJA_ENV
+import constants.cdr_cleaner.clean_cdr as cdr_consts
+from cdr_cleaner.cleaning_rules.deid.concept_suppression import \
+    AbstractBqLookupTableConceptSuppression
+
```

Comment:
for logging unity, add `from utils import pipeline_logging` and the configuration below